test/binaryen.js/fast-math.txt

```diff
@@ -0,0 +1 @@
+// fastMath=true 
```

Comment:
Wouldn't printing the default result in `false` here?

Reply:
it should. I just manually created this fixture artefact) Thanks again